apps/web/public/static/locales/fr/common.json

```diff
@@ -1620,7 +1620,7 @@
   "form_deleted": "Formulaire supprimé",
   "delete_form": "Supprimer le formulaire",
   "delete_form_action": "Oui, supprimer le formulaire",
-  "delete_form_confirmation": "Voulez-vous vraiment supprimer ce formulaire ? Toute personne avec qui vous avez partagé le lien ne pourra plus passer de réservation en l'utilisant. De plus, toutes les réponses associées seraient supprimées.",
+  "delete_form_confirmation_1": "Voulez-vous vraiment supprimer ce formulaire ? Toute personne avec qui vous avez partagé le lien ne pourra plus passer de réservation en l'utilisant. De plus, toutes les réponses associées seraient supprimées.",
```

Comment:
why this change? Here we are not breaking the string in 2 parts and just assigning the complete string to confirmaton_1